src/pytest_split/plugin.py

```diff
@@ -134,4 +134,23 @@ def _calculate_suite_start_and_end_idx(splits, group, items, stored_durations):
     if not end_idx:
         end_idx = len(items)
 
+    # Modify start_idx and end_idx so they match start/end of an ipynb file
+    if _is_ipy_notebook(item_node_ids[start_idx]):
```

Comment:
Could you please add some test case for this whole notebook thing?

Reply:
Have a look at test_plugin.py. I think you should be able to create some simple test notebook with `testdir.makefile` and for the rest there should be good examples already in the existing tests.

Reply:
@jerry-git I added `test_ipynb.py` which is basically almost the same as `test_plugin.py`, but for notebooks. I tried merging it with `test_plugin.py` but it's my first time working with `pytest` fixtures, and I'm still trying to wrap my head around it! Also, for some reason, running pytest on `test_plugin.py` and `test_ipynb.py` individually works fine, but it fails when invoking it on both, i.e. via running simply `pytest`. Could you please take a look? Thanks.